web/README.md

```diff
@@ -16,13 +16,23 @@ PAG Web SDK is built on WebAssembly and WebGL which supports all of the PAG feat
 
 PAG Web SDK consists of two files: `libpag.js` and `libpag.wasm`.
 
-You can use the `locateFile` function to get the path of `libpag.wasm` file. By default, the `libpag.wasm` file is located next to the `libpag.js` file.
-
 ### Browser (Recommend)
 
+Direct `<script>` Include, `libpag` will be registered as a global variable.
+
+For production, we recommend link to a specific version number and build to avoid unexpected breakage from newer versions:
```

Comment:
For production use, we recommend using a specific version number of libpag to avoid unexpected breakage from newer versions: